creusot/tests/should_succeed/bdd/why3session.xml

```diff
@@ -40,12 +40,35 @@
   <goal name="vc_hash.2" expl="postcondition" proved="true">
   <proof prover="6"><result status="valid" time="0.606420" steps="47937"/></proof>
   </goal>
+  <goal name="vc_hash.3" proved="true">
+  <transf name="destruct_term_subst" proved="true" arg1="childf">
```

Comment:
annoyingly this proof requires a manual destruct, I suspect triggers / data type encoding are to blame.